recipe/meta.yaml

```diff
@@ -212,6 +212,8 @@ outputs:
     script: install-symlinks.sh
     build:
       skip: true  # [target_platform != cross_target_platform]
+      track_features:            # [not with_specs]
+        - gcc_no_conda_specs     # [not with_specs]
```

Comment:
You're going to need a selector (`# [with_specs]`) for
https://github.com/conda-forge/ctng-compilers-feedstock/blob/7449167b23174fc2ae7518a9efc306ce1226f901/recipe/meta.yaml#L220
as well

Reply:
I don't see how `gcc-no-conda-specs` can run-depend on `gcc` (with a transitive run-requirement on `conda-gcc-specs`) while also having a `conda-gcc-specs <0.0a0` constraint.